n2y/blocks.py

```diff
@@ -570,8 +570,30 @@ def to_pandoc(self):
         return self.children_to_pandoc()
 
 
-class LinkToPageBlock(WarningBlock):
-    pass
+class LinkToPageBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
+
+        self.link_type = self.notion_data["type"]
+        # The key for the object id may be either "page_id"
+        # or "database_id".
+        self.linked_page_id = self.notion_data[self.link_type]
+
+    def to_pandoc(self):
+        # TODO: in the future, if we are exporting the linked page too, then add
+        # a link to the page. For now, we just display the text of the page.
+
+        # The linked page may be outside the scope of what is accessible to n2y.
+        try:
+            page = self.client.get_page_or_database(self.linked_page_id)
+            title = page.title.to_pandoc()
+        except PermissionError:
```

Comment:
Does `client.get_page_or_database` throw a `PermissionError` when the page is not accessible?